fastlane/Fastfile

```diff
@@ -45,6 +46,18 @@ platform :ios do
   end
 
   lane :coverage do
+    slather(
+      output_directory: './artifacts/coverage',
+      scheme: ENV['REM_FL_TESTS_SCHEME'] || 'Tests',
+      sonarqube_xml: true,
+      use_bundle_exec: true,
+      proj: ENV['REM_FL_TESTS_PROJECT'],
+      workspace: ENV['REM_FL_TESTS_WORKSPACE'],
+      binary_basename: ENV['REM_FL_TESTS_SLATHER_BASENAME'],
+      ignore: '*.{h,m}')
+  end
+
+  lane :run_xcov do
```

Comment:
Does this still get used?

Reply:
No - it's just for devs to be run locally if needed. I might as well delete this lane